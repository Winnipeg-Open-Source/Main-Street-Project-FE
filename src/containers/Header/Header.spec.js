import React from 'react';
import renderWithTheme from 'tests/helpers/renderWithTheme';
import HeaderContainer from './Header';

describe('HeaderContainer', () => {
    it('renders without crashing', () => {
        const { asFragment } = renderWithTheme(<HeaderContainer />);

        expect(asFragment()).toMatchSnapshot();
    });
});
