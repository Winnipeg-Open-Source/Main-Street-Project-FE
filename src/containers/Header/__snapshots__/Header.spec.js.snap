// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderContainer renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm sc-AxiKw sc-pkfPC iGeEdY"
    color="primary"
  >
    <svg
      aria-hidden="true"
      class="sc-Axmtr dFmDHs sc-pAZqv sc-psDhf gTeQdV"
      data-testid="menu-icon"
      fill="currentcolor"
      focusable="false"
      height="24"
      role="img"
      tabindex="-1"
      title="Menu"
      viewBox="0 0 24 24"
      width="24"
    >
      <title>
        Menu
      </title>
      <path
        d="M3 13h18v-2H3v2zm0 5h18v-2H3v2zM3 6v2h18V6H3z"
      />
    </svg>
    Main Street Project Inventory System
  </div>
</DocumentFragment>
`;
